microsite/src/main/tut/overview/index.md

```diff
@@ -1,128 +1,47 @@
- * **Composability**. Purely functional code can't be combined with impure code that has side-effects without sacrificing the straightforward reasoning properties of functional programming. `ZIO` lets you wrap up all effects into a purely functional package that lets you build composable real world programs.
- * **Concurrency**. `ZIO` has all the concurrency features of `Future`, and more, based on a clean fiber concurrency model designed to scale well past the limits of native threads. `ZIO`'s concurrency primitives do not leak resources.
- * **Interruptibility**. All concurrent computations can be interrupted, in a way that still guarantees resources are cleaned up safely, allowing you to write aggressively parallel code that doesn't waste valuable resources or bring down production servers.
- * **Resource Safety**. `ZIO` provides composable resource-safe primitives that ensure resources like threads, sockets, and file handles are not leaked, which allows you to build long-running, robust applications. These applications will not leak resources, even in the presence of errors or interruption.
- * **Testability**. `ZIO` allows you to declare dependencies for your code (like database, configuration, or web service) in a compositional, type-safe way, and easily inject both live and test versions, so you can build fast, deterministic unit tests that give you confidence to refactor safely.
- * **Immutability**. `ZIO`, like Scala's immutable collection types, is an immutable data structure. All `ZIO` methods and functions return new `ZIO` values. This lets you reason about `ZIO` values the same way you reason about immutable collections.
- * **Reification**. `ZIO` reifies programs. In non-functional Scala programming, you cannot pass programs around or store them in data structures, because programs are not values. But `ZIO` turns your programs into ordinary values, and lets you pass them around and compose them with ease.
- * **Performance**. Although simple, synchronous `ZIO` programs tend to be slower than the equivalent imperative Scala, `ZIO` is extremely fast given all the expressive features and strong guarantees it provides. Ordinary imperative Scala could not provide these features and performance without tedious, error-prone boilerplate that no one would ever write.
+Finally, if you are an experienced functional programmer, then direct use of the `ZIO` data type is recommended, although you may find it useful to create your own family of type aliases in different parts of your application.
 
-Nearly all programmers will find the features of `ZIO` help them build scalable, performant, concurrent, and leak-free applications faster and with stronger correctness guarantees than legacy techniques allow.
+# Creating ZIO Values
 
+The simplest ZIO effects you can create model success and failure. These are covered in the [next session](success_failure.html).
```

Comment:
Broken link here? Should link to `creating_effects.md`

Reply:
Fixed! 🙏